src/styles/components/ec-card/ec-card.css

```diff
@@ -0,0 +1,38 @@
+:root {
```

Comment:
I wouldn't create variables for this. We can use TW to make something reusable:
```
.ec-card-transition {
  @apply tw-ease tw-duration-400;
}

.ec-card {
  &--is-interactive {
    @apply tw-transition-transform ec-card-transition;  

    &:after {
      @apply tw-transition-opacity ec-card-transition;
    }
  }
}
```
the class `.ec-card-transition` will be purged by purgecss.
the class `tw-duration-400` doesn't exist but we can add it in tailwind.config.js:
```
transitionDuration: {
      ...
      300: '300ms',
      400: '400ms',
      500: '500ms',
      ...
    },
```

Reply:
Wow, much better, thx!